Documentation/HowToGuides/UsingCombinedActions/README.md

```diff
@@ -0,0 +1,205 @@
+
+![Add Position Mutator](assets/images/AddPositionMutator.png)
+
+### Step 7
+
+Drag and drop the `Capsule` GameObject into the `Target` parameter on the `Transform Position Mutator` Facade.
+
+![Drag And Drop Capsule](assets/images/DragAndDropCapsule.png)
+
+### Step 8
+
+Expand the `Tilia.Input.CombinedActions.Unity` Package directory in the Unity Project window and select the `Packages -> Tilia.Input.CombinedActions.Unity -> Runtime -> Prefabs` directory then drag and drop the `Input.CombinedActions.AxesToVector3Action` prefab into the Unity hierarchy window.
+
+![Import Axes To Vector3 Action](assets/images/ImportAxesToVector3Action.png)
+
+### Step 9
+
+On the `Input.CombinedActions.AxesToVector3Action` GameObject, Drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Horizontal` GameObject into the `Lateral Axis` property on the `Axes To Vector3` component.
+
+![Lateral Axis](assets/images/LateralAxis.png)
+
+and drag and drop the `Input.CombinedActions.BooleanTo1DAxisAction Vertical` GameObject into the `Longitudinal Axis` property on the `Axes To Vector3` component.
+
+![Longitudinal Axis](assets/images/LongitudinalAxis.png)
+
+### Step 10
+
+Select the `Input.CombinedActions.AxesToVector3Action` GameObject from the Unity Hierarchy and click the `+` symbol in the bottom right corner of the `Value Changed` event parameter on the `Axes To Vector3` component.
+
+Drag and drop the `Movement` GameObject into the event listener box that appears on the `Value Changed` event parameter on the `Axes To Vector3` component that displays `None (Object)`.
+
+![Drag Movement Into Event Listener](assets/images/DragMovementIntoEventListener.png)
+
+Select a function to perform when the `Value Changed` event is emitted. For this example, select the `TransformPositionMutator` -> `DoIncrementProperty` function (be sure to select `Dynamic Vector3 - DoIncrementProperty` for this example).
+
+![Set Value Changed Function](assets/images/SetValueChangedFunction.png)
+
+> Play the Unity scene you should now be able to control the GameObject using the A, S, D, W keys.
+
+![Move Capsule](assets/images/MoveCapsule.png)
+
+### Step 11
+
+We are now going to make it possible to rotate the capsule instead. 
+
+Disable the `Movement` GameObject.
+
+![Disable Movement GameObject](assets/images/DisableMovementGameObject.png)
+
+### Step 12
+
+Create a new `Empty` Unity Object by selecting `Main Menu -> GameObject -> Create Empty` and rename it `Rotation`. Click `Add Component` and add a `Float To Vector3` component.
+
+![Add Float To Vector3](assets/images/AddFloatToVector3.png)
+
+Click `Add Component` again and add a `Transform Euler Rotation Mutator` component.
+
+![Add Euler Rotator](assets/images/AddEulerRotator.png)
+
+### Step 13
+
+Drag and drop the `Capsule` GameObject into the `Target` parameter on the `Transform Euler Rotation Mutator` Facade. On the `Mutate on X` property turn off the `X` and `Z`  values.
```

Comment:
`Facade` should be `component` so:

`Transform Euler Rotation Mutator` component.

Also it should be the `Mutate on Axis` property.

Reply:
Maybe worth sticking a note under here to explain why the x and z axis are turned off, could read like this:

```
### Step 13

Drag and drop the `Capsule` GameObject into the `Target` parameter on the `Transform Euler Rotation Mutator` component. On the `Mutate on Axis` property turn off the `X` and `Z`  checkboxes.

> The reason we only want the Y axis turned on is because we only want to rotate around this axis to make our capsule rotate to a new facing direction in the scene.
```